### PR TITLE
[DEV] Lift numba version to 0.49

### PR DESCRIPTION
**Pull request by StrayBird-ATSH**
## Description ##
Currently the CI is failed due to incompatible versions between numba and llvm. 
This pull request fixes this problem

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

File: env/gpu/py3.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ dependencies:
     - nbsphinx>=0.3.4,<0.4
     - ipython
     - ipykernel
-    - numba==0.47
+    - numba==0.49
```
Comment (leezu) [inline env/gpu/py3.yml:19]: Would it be required to specify as 0.49.0?

**Comment by mli** [comment]
Job PR-1208/9 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1208/9/index.html
